frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
 local _ = require("gettext")
 
 local QuickStart = {
-    quickstart_force_show_version = 201511982,
+    quickstart_force_show_version = 2021070000,
 }
 
 local language = G_reader_settings:readSetting("language") or "en"
 local version = Version:getNormalizedCurrentVersion()
 local rev = Version:getCurrentRevision()
 
+local stylesheet = [[
+div.logo { float: right; }
+div.logo > img { height: 4em; }
+h1 { page-break-before: avoid; margin-bottom: 1em; }
+h2 { background-color: black; color: white; text-align: center; page-break-after: avoid; }
+
+hr { margin: 1em 20%; }
+div.generated { font-size: x-small; }
+
+li { margin: 0.5em 0; page-break-inside: avoid; }
+blockquote { text-align: center; page-break-before: avoid; page-break-inside: avoid; } /* Markdown lines starting with '>' */
+
+/* Inline image (icons) */
+p img, blockquote img { height: 1.5em; vertical-align: bottom; }
+
+/* Standalone image (UI screenshots) */
+div.img-block { text-align: center; margin: 0.5em; }
+div.img-block > img { max-height: 60vh; max-width: 80vw; }
+div.break-before-avoid { page-break-before: avoid; }
+div.break-after-avoid { page-break-after: avoid; }
+
+div.table { display: table; page-break-inside: avoid; width: 100%; }
+div.table > div { display: table-row; }
+div.table > div > * { display: table-cell; text-indent: 0; padding: 0.3em; }
+div.table > div > *:nth-child(2) { text-align: left; hyphens: none; background-color: #eeeeee; }
+div.table > div > *:nth-child(3) { white-space: nowrap; }
+]]
+
 local quickstart_guide = T(_([[
-# KOReader Quickstart Guide
+<div class="logo">![KOReader](../resources/koreader.svg)</div>
+
+# QUICKSTART GUIDE
+
+* [USER INTERFACE](#ui)
+* [USER INTERFACE TIPS](#uitips)
+* [ACCESSING FILES](#afiles)
+* [TRANSFERRING FILES](#tfiles)
+* [FRONTLIGHT/BACKLIGHT](#flight)
+* [WHILE READING](#reading)
+* [INSTALLING DICTIONARIES](#dicts)
+* [MORE INFO](#more)
+
+---
+
+You can access the complete user manual from [our GitHub page](https://github.com/koreader/koreader).
+
+## USER INTERFACE <a id="ui"></a>
```

Comment:
Also I'd suggest not to bother translators with this. _("User interface") .. '<a id=""ui"></a>' will do just fine.

As an aside, I'm not sure if `## User interface {#ui}` is standardized but we could potentially add that to the markdown generator for better results. (There have been a few minor updates to it in the past couple of years.)

Reply:
`_("User interface") .. '<a id=""ui">' `

This feels it would fragment the text to translate even more (are we sure weblate will present all these fragments in the correct order to give some context ?)
And if we want the heading to be the same as in the TOC, we'd need to re-use these these bits in this TOC:
```
_(# Quickstart guide)..[[

* []] .. _("User interface") .. [[](#ui)
* []] .. _("User interface tips") .. [[](#uitips)
* []] .. _("Accessing files") .. [[](#afiles)        oh my, I can't go on!
* [Transferring files](#tfiles)
* [Frontlight/backlight](#flight)
* [While reading](#reading)
* [Installing dictionaries](#dicts)
* [More info](#more)
```

Reply:
I would like to fragment it per paragraph à la Markdown, including separate headings. I'm only asking for sections because it'd be a PITA to write without automation.

And yes, that's in part because what you describe above could then happen automagically.

> This feels it would fragment the text to translate even more (are we sure weblate will present all these fragments in the correct order to give some context ?)

Yes, they will be in order.

Reply:
It's quite annoying dealing with units longer than about a paragraph while translating. Translating several paragraphs at once you have to either finish it all at once or mark it as dirty, it's an absolute pain with small changes, etc.